src/dojos/cd-dojo-details.vue

```diff
@@ -106,7 +106,17 @@
     },
     computed: {
       address() {
-        return !this.dojoDetails.address1 ? undefined : [
+        if (!this.dojoDetails.address1) {
+          return undefined;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.countryName)) {
+          return this.dojoDetails.address1;
+        } else if (this.dojoDetails.address1.includes(this.dojoDetails.placeName)) {
+          return [
+            this.dojoDetails.address1,
+            this.dojoDetails.countryName,
+          ].join(', ');
+        }
+        return [
```

Comment:
My thoughts here are more like : either use the address or the placeName, preferably the address.
I don't think there is need for such extensive checks